includes/classes/Templates.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Template functions for Orbit.
+ *
+ * @package         Orbit
+ */
+
+namespace Eighteen73\Orbit;
+
+/**
+ * This class is built upon BE Media from Production so all due credit to those authors.
+ * http://www.github.com/billerickson/be-media-from-production
+ */
+class Templates {
+
+	use Environment;
+	use Singleton;
+
+	/**
+	 * Function to get the template file from the theme if it exists, otherwise from the plugin.
+	 *
+	 * @param string $template_name The name of the template file.
+	 * @return string The path to the template file.
+	 */
+	private static function get_template_part( $template_name ) {
+		$theme_template = get_stylesheet_directory() . '/template-parts/' . $template_name;
```

Comment:
Update to https://developer.wordpress.org/reference/functions/get_theme_file_path/

I think the theme path to check should be `orbit/branded-emails` so we can extend orbit with additional templates in the future, and matches WooCommerce with a plugin name in the root directory.